dist/profile/manifests/archives.pp

```diff
@@ -2,18 +2,72 @@
 # Defines an archive server for serving all the archived historical releases
 #
 class profile::archives (
-    Array  $rsync_hosts_allow       = ['localhost'],
-    String $archives_dir            = '/srv/releases',
-    String $rsync_motd_file         = '/etc/jenkins.motd',
-    String $source_mirror_endpoint  = 'ftp-osl.osuosl.org',
-    String $source_mirror_directory = '/jenkins/',
-    String $owner                   = 'www-data',
-    String $group                   = 'www-data',
+    Array  $rsync_hosts_allow           = ['localhost'],
+    String $archives_dir                = '/srv/releases',
+    String $rsync_motd_file             = '/etc/jenkins.motd',
+    String $source_mirror_endpoint      = 'ftp-osl.osuosl.org',
+    String $source_mirror_directory     = '/jenkins/',
+    Array  $ssh_authorized_keys         = [],
   ) {
   include ::stdlib
   include profile::apachemisc
   include profile::letsencrypt
 
+  $apache_owner     = 'www-data'
+  $apache_group     = $apache_owner
+
+  ## Manage mirrorsync user and its home directory
+  user { 'mirrorsync':
+    ensure     => present,
+    shell      => '/bin/bash',
+    managehome => true,
+  }
```

Comment:
Should this user associated to the `www-data` (`$apache_group`) group? Or the other way: `www-data` should be added to the group `mirrorsync` ?

Reply:
I don't think the user should be associated with "www-data", the only think that the user "mirrorsync" should do is being allowed to run the script "mirrorsync" which then run a rsync command as the user www-data